CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to React Native Sample APP
+
+Please follow this guidelines to start contributing to this project.
+
+# Creating a pull request
+
+1. Create a new branch from **main** branch following the name convention.
+2. Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) convention for commit messages.
+3. Create a PR and include any useful context of the change in the description. Include any relevant screenshots.
+4. Wait for the PR to be reviewed and approved.
+
+# Branch naming convention
+
+The branch name should include the category, the Jira ticket number and a short description using "-" as separator.
+
+Use the following format: {category}/{Jira tickket number}-{short description}.
```

Comment:
You got one extra "k"  :)

Reply:
good catch!

Reply:
Is this meant for people outside the CSQ to contribute?  If so, they will not have access to the JIRA board.